docs/conf.py

```diff
@@ -10,8 +10,9 @@
 # General --------------------------------------------------------------
 
 master_doc = "index"
-extensions = ["sphinx.ext.autodoc", "sphinx.ext.intersphinx", "pallets_sphinx_themes"]
+extensions = ["sphinx.ext.autodoc", "sphinx.ext.intersphinx", "sphinx.ext.autosectionlabel", "pallets_sphinx_themes"]
```

Comment:
Since most headings in the Click docs (eg. "Confirmation prompts" in https://raw.githubusercontent.com/pallets/click/master/docs/prompts.rst) have no sphinx labels, then the headings don't show up in the sphinx inventory file, and hence aren't picked up by doc2dash and aren't searchable.

https://stackoverflow.com/questions/5874796/sphinx-automatically-label-headings